lib/functionTemplates/py2/.gitignore

```diff
@@ -0,0 +1,2 @@
+*.py[doc]
```

Comment:
Since we only currently copy `binaris.yml` and the function file it's ok, but if we'll switch to directory copy, this may hurt the users. Consider keeping `.gitignore` in the `functionTemplates` directory with `*/.binaris` rule for example (same goes for the `node8` template)

Reply:
I just copied from the node template, is the .gitignore there by mistake?

Reply:
It wasn't an issue before because we had a single runtime. I think @michaeladda is suggesting to unify things so `binaris.yml` is handled in a similar fashion to the `.gitignore`.